src/common/Accessibility.ts

```diff
@@ -13,6 +13,10 @@ import RX = require('../common/Interfaces');
 
 export abstract class Accessibility extends RX.Accessibility {
     abstract isScreenReaderEnabled(): boolean;
+    
+    isHighContrastEnabled(): boolean|undefined {
```

Comment:
>boolean|undefined [](start = 29, length = 17)

Do you think it's valuable to include `undefined` in the set of possible values?

What do you think of the idea of using `false` for platforms that don't support querying their high contrast setting?

Reply:
Then if iOS or Android invokes `RX.Accessibility.isHighContrastEnabled()` and it returns `false` it won't be able to tell whether high contrast is disabled or not supported., while the `undefined` value says "don't know."